docs2/site/docs/migrations/migration8.md

```diff
@@ -1602,6 +1602,41 @@ for reference.
 
 If you directly implement `IResolveFieldContext`, you must now also implement the `ExecutionContext` property.
 
+### 30. `GraphType.Initialize` method is now called after initialization is complete
+
+The `Initialize` method on each `GraphType` is now called after the schema has been fully initialized. As such,
+you cannot add fields to the graph type expecting `SchemaTypes` to resolve types and apply name converters.
+If it is necessary for your graph type to add fields dynamically, you should do so in the constructor or else
+set the `ResolvedType` property for the new fields. Failing to do so will result in a schema validation exception.
+
+Please note that the constructor is the preferred place to add fields to a graph type.
+
+```csharp
+// v7
+public class MyGraphType : ObjectGraphType
+{
+    public override void Initialize(ISchema schema)
+    {
+        AddField(new FieldType {
+            Name = "Field",
+            Type = typeof(StringGraphType)
+        });
+    }
+}
+
+// v8
+public class MyGraphType : ObjectGraphType
+{
+    public override void Initialize(ISchema schema)
+    {
+        AddField(new FieldType {
+            Name = "field",
```

Comment:
```suggestion
            Name = "Field",
```

Reply:
@Shane32 just to confirm, does this mean we no longer need to specify Type when ResolvedType is set?

Reply:
That’s correct

Reply:
I don't think it demonstrate much but could lead to questioning. I suggest mentioned the name converter in the document or add a comment explaining the reasoning behind the field name change.

Reply:
```suggestion
            Name = "field", // name converter is not applied anymore, so the name must be exactly as desired
```